lib/signaling/track.js

```diff
@@ -70,7 +58,7 @@ class TrackSignaling extends EventEmitter {
       trackTransceiver: {
         enumerable: true,
         get() {
-          return trackTransceiver;
+          return this._trackTransceiver;
         }
```

Comment:
Nitpick: We could convert this to a `get trackTransceiver() { return this._trackTransciever; }` on the `class`.